pv_site_production/app.py

```diff
@@ -2,25 +2,81 @@
 Apply the model to the PVs in the database and note the results.
 """
 
+import datetime as dt
 import logging
 import os
 import pathlib
-from datetime import datetime
+from uuid import UUID
 
 import click
 import dotenv
 from psp.models.base import PvSiteModel
+from psp.typings import PvId, Timestamp, X
 from pvsite_datamodel.connection import DatabaseConnection
-from pvsite_datamodel.write import insert_forecast_values
+from pvsite_datamodel.sqlmodels import ForecastSQL, ForecastValueSQL
 
 from pv_site_production.data.pv_data_sources import DbPvDataSource
-from pv_site_production.models.common import apply_model
 from pv_site_production.utils.config import load_config
 from pv_site_production.utils.imports import import_from_module
 
 _log = logging.getLogger(__name__)
 
 
+def _do_one_pv(
```

Comment:
Could you add a more specific name, not quite sure what 'do' is here. Like `run_model_and_save_for_one_pv`

Would be great to add function description too